src/clients/db/PatientRepository.ts

```diff
@@ -13,12 +13,13 @@ export class PatientRepository extends Repository<Patient> {
   }
 
   async search(text: string): Promise<Patient[]> {
+    const cleanText = text.replace(/[\\[()?*]+/g, '')
```

Comment:
I also checked with String ```{1}```.
```SyntaxError: Invalid regular expression: /{1}/: Nothing to repeat.```
Need to handle this case as well.

Reply:
I think that we should also remove `]`, and `{`, and `}`

Reply:
Ok, will do.

Reply:
I think that we need to remove all valid regex chars. Is there any lib out there that can help us to not forget any char?